bitnami/keycloak/21/debian-11/prebuildfs/opt/bitnami/scripts/libvalidations.sh

```diff
@@ -207,6 +229,28 @@ validate_ipv4() {
     return $stat
 }
 
+########################
+# Validate if the provided argument is a valid IPv6 address
+# Arguments:
+#   $1 - IP to validate
+# Returns:
+#   Boolean
+#########################
+validate_ipv6() {
+    local ip="${1:?ip is missing}"
+    local stat=1
+
+    local full_address_regex='^([0-9a-fA-F]{1,4}:){7}[0-9a-fA-F]{1,4}$'
+		local short_address_regex='^((([0-9a-fA-F]{1,4}:){0,6}[0-9a-fA-F]{1,4}){0,6}::(([0-9a-fA-F]{1,4}:){0,6}[0-9a-fA-F]{1,4}){0,6})$'
+
+
+    if [[ $ip =~ $full_address_regex || $ip =~ $short_address_regex || $ip == "::" ]]; then
+        stat=0
+    fi
+
```

Comment:
And also please remove these extra lines as well, just to be consistent with `validate_ipv4()`